content/docs/rfcs/48/README.md

```diff
@@ -0,0 +1,105 @@
+---
+slug: 48
+title: 48/RLN-INTEREP-SPEC
+name: Interep as group management for RLN
+status: raw
+category:  
+tags: rln
+editor: Aaryamann Challani <aaryamann@status.im>
+contributors:
+---
+
+# Abstract
+
+This spec integrates [Interep](https://interep.link) into the [RLN](https://rfc.vac.dev/spec/32/) spec. 
```

Comment:
In abstract I'd mention motivation briefly, e.g. a one sentence summary of Motivation section (~ allow users to join groups w/o funds but bootstrapping with web2 identity)

Reply:
Addressed in [9de124a](https://github.com/vacp2p/rfc/pull/564/commits/9de124addb6e16f5ec6908336a9d26e29b056f7e)